python/python/lance/fragment.py

```diff
@@ -321,11 +324,11 @@ def to_table(
             with_row_id=with_row_id,
         ).to_table()
 
-    def add_columns(
+    def merge_columns(
```

Comment:
Copy the docstring from `add_columns`?

Reply:
It is copied, and modified slightly for the new return value. This function does the same things as `add_columns`, but it has a slightly different return value. This is my way of fixing the API without introducing a breaking change. Not sure if this answers what you are asking, though.